### PR TITLE
Call TMDb Person API with metadata language

### LINKED ISSUE #6015: Actor metadata is not downloaded in the specified language
**Describe the bug**
All the actor biography in my jellyfin server are in English. However, I configured the metadata download language to french, the actor is correctly link to the TMDB ID, and french biography exist in the TMDB actor page. If I try to manually identify the actor with the TMDB ID, the problem remains the same.

**System (please complete the following information):**
 - OS: Ubuntu 20.04.2 LTS
 - Virtualization: N/A
 - Clients: Browser
 - Browser: Google Chrome 90.0.4430.93 (Official build) (64 bits)
 - Jellyfin Version: 10.7.5
 - Installed Plugins: none
 - Reverse Proxy: none
 - Base URL: none
 - Networking: Host
 - Storage: local

**To Reproduce**

1. Configure "Preferred display language" to French
2. Configure "Display language" to French
3. Configure "Preferred Metadata Language" to French
4. Configure in the Library Settings the "Preferred download language" to French
5. Manually identify an actor with the TMDB ID that has a french biography on TMDB
6. Note that the biography is still in English
P.s. I have done the steps 1 to 4 before creating my first library

**Expected behavior**
That Jellyfin retrieves the biography in the specified metadata language

**Logs**
[2021-05-08 17:46:28.572 +00:00] [INF] Setting provider id's to item 4783d0ca-394f-eaa4-fb94-0ea7c3d29378-"Sylvester Stallone": "{\"Tmdb\":\"16483\"}"

**Screenshots**
image
image
image
image
image

### PR DESCRIPTION
**Pull request by Ullmie02**
**Changes**
Persons always had their English biography instead of the localized one. 
I also changed the `NormalizeLanguage` method because the TMDb API doesn't support de-CH, fr-CH and it-CH, so it now uses the language without the country code.

**Issues**
Fixes #6015

### CODE REVIEW

**Review by crobibero — APPROVED** [review]

**Review by deltonio2 — APPROVED** [review]

File: MediaBrowser.Providers/Plugins/Tmdb/People/TmdbPersonImageProvider.cs [hunk 1]
```diff
@@ -48,21 +47,21 @@ public IEnumerable<ImageType> GetSupportedImages(BaseItem item)
 
         public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, CancellationToken cancellationToken)
         {
+            var language = item.GetPreferredMetadataLanguage();
```
Comment (Bond-009) [inline MediaBrowser.Providers/Plugins/Tmdb/People/TmdbPersonImageProvider.cs:50]: Move this to line 52

File: MediaBrowser.Providers/Plugins/Tmdb/TmdbUtils.cs [hunk 2]
```diff
@@ -148,6 +148,12 @@ public static string NormalizeLanguage(string language)
 
             if (parts.Length == 2)
             {
+                // TMDB doesn't support Switzerland (de-CH, it-CH or fr-CH) so use the language (de, it or fr) without country code
+                if (string.Equals(parts[1], "ch", StringComparison.Ordinal))
```
Comment (Bond-009) [inline MediaBrowser.Providers/Plugins/Tmdb/TmdbUtils.cs:152]: Are we sure this is always lowercase?
And maybe add a test for this

**Review by Bond-009 — APPROVED** [review]

File: MediaBrowser.Providers/Plugins/Tmdb/TmdbUtils.cs (hunk 2 above)
Comment (Ullmie02) [reply MediaBrowser.Providers/Plugins/Tmdb/TmdbUtils.cs]: Sorry I was confused. In theory it should always be upper case but to be safe I changed it to ignore case

File: MediaBrowser.Providers/Plugins/Tmdb/People/TmdbPersonImageProvider.cs [hunk 3]
```diff
@@ -49,20 +48,20 @@ public IEnumerable<ImageType> GetSupportedImages(BaseItem item)
         public async Task<IEnumerable<RemoteImageInfo>> GetImages(BaseItem item, CancellationToken cancellationToken)
         {
             var person = (Person)item;
+            var language = item.GetPreferredMetadataLanguage();
```
Comment (Bond-009) [inline MediaBrowser.Providers/Plugins/Tmdb/People/TmdbPersonImageProvider.cs:51]: Move to line 57, no need to call this if we do an early return

**Review by Bond-009 — APPROVED** [review]

**Pull request merged**